opendbc/car/tesla/values.py

```diff
@@ -85,20 +85,20 @@ class CANBUS:
 
 
 class CarControllerParams:
-  ANGLE_LIMITS: AngleSteeringLimits = AngleSteeringLimits(
-    # EPAS faults above this angle
-    360,  # deg
-    # Tesla uses a vehicle model instead, check carcontroller.py for details
-    ([], []),
-    ([], []),
-  )
-
   STEER_STEP = 2  # Angle command is sent at 50 Hz
   ACCEL_MAX = 2.0    # m/s^2
   ACCEL_MIN = -3.48  # m/s^2
   JERK_LIMIT_MAX = 4.9  # m/s^3, ACC faults at 5.0
   JERK_LIMIT_MIN = -4.9  # m/s^3, ACC faults at 5.0
 
+  ANGLE_LIMITS: AngleSteeringLimits = AngleSteeringLimits(
+    360,  # deg, EPAS faults above this angle
+    # Tesla uses a vehicle model instead, check carcontroller.py for details
+    ([], []),
+    ([], []),
+    (1 / (DT_CTRL * STEER_STEP)),  # 50 Hz
```

Comment:
this is redundant information

Reply:
We can: 

1. Simply put 50 there as value instead of the calculation (I prefer that, but kept it as you had it) 
2. We can make a new `AngleSteeringLimits` named something like `VmAngleSteeringLimits` with only the relevant parts for this control scheme?
3. Make the caller provide it independently on every call to the methods instead of bundled as the `AngleSteeringLimits`?

Let me know which one should we do or if you had something else in mind @sshane

Reply:
well I've made it just "50" for tesla. I need this so that I can pass it down to the angle steering calulation without having to add infinite amount of arguments to the methods